lab_6_pipeline/pipeline.py

```diff
@@ -9,6 +9,19 @@
 
```

Comment:
keep up

you can still complete lab 6 till Friday

Reply:
P.S. Please tell me how to make tests run for lab 6 and not lab 5? I changed target_score for lab 6, but it didn't work

Reply:
Please tell me how to make the tests run for lab 6 and not lab 5? I changed target_score for lab 6, but it didn't work